src/RAWcookedBundle/Resources/views/Default/index.html.twig

```diff
@@ -106,40 +106,40 @@
+<p>The Matroska files that have been encoded by <i>RAWcooked</i> can be used with many popular open source softwares which support FFV1 and FLAC playback, including:</p>
 <ul>
-<li><a href="https://www.videolan.org/vlc/">VLC</a> media player for playback</li>
-<li><a href="https://www.ffmpeg.org/">FFmpeg</a> tools can transcode, playback and probe those files</li>
-<li><a href="{{ path('qc_home') }}">QCTools</a> can performs in-depth analysis</li>
-<li><a href="{{ path('mi_home') }}">MediaInfo</a> can display technical information</li>
-<li><a href="{{ path('mc_home') }}">MediaConch</a> can check the conformance of the file</li>
+<li><a href="https://www.videolan.org/vlc/">VLC</a> media player for quick and simple review</li>
+<li><a href="https://www.ffmpeg.org/">FFmpeg</a> for transcoding, playback, metadata viewing and extraction</li>
+<li><a href="{{ path('qc_home') }}">QCTools</a> advanced quality control software for audio-visual files</li>
+<li><a href="{{ path('mi_home') }}">MediaInfo</a> display and export technical metadata</li>
+<li><a href="{{ path('mc_home') }}">MediaConch</a> for checking the conformance of a file</li>
 </ul>
-<p>This way an archive with limited resources can process the files encoded by <i>RAWcooked</i> on every current computer (e.g. for cataloging or access purposes).</p>
+<p>Being open source, they’re free to use allowing archives with limited resources to process the files encoded by <i>RAWcooked</i> on any computer, such as for cataloging or access purposes.</p>
 <h2>Funding plan</h2>
-<p>MediaArea.net realizes the implementation of the basis module for anticipated costs of 10&nbsp;000&nbsp;&euro; which are guaranteed by AV Preservation by reto.ch.</p>
-<p>The archival community and the industry are invited to <a href="{{ path('ma_contact') }}">buy</a> the flavors (components, bit depths, endianess, packing, etc.) of the formats they actually need. An example file of each flavor of a format will be tested by MediaArea.net and the exact specification implemented into the software.</p>
-<p>This way the contributing bodies and individuals are sure their files can be processed by the software.</p>
-<p>The price is 1&nbsp;000&nbsp;&euro; for basic support (bug fixes) and one non-free flavour, then 500&nbsp;&euro; for each following non-free flavour (additional flavours can be added later if you are not sure of how many flavours you need now, at the same price).</p>
+<p>MediaArea.net has achieved implementation of RAWcooked’s basic module with thanks to AV Preservation reto.ch, who has guaranteed software development with funding of 10,000€.</p>
+<p>The archive community or interested industry users are invited to <a href="{{ path('ma_contact') }}">buy</a> additional flavours required by their institution - such as components, bit depths, endianess, packing, etc.  Required flavours will be fully tested by MediaArea.net to exact specification before being implemented into the software.</p>

```

Comment:
Actually they are already implemented.
So misleading.
Something like:
"The archive community or interested industry users are invited to \<a href="{{ path('ma_contact') }}">buy\</a> additional flavours required by their institution - such as components, bit depths, endianess, packing, etc in order to help the long term sustainability of the project and further development ."